stunnel/map.jinja

```diff
@@ -15,12 +15,15 @@
     }
 }, merge=True)%}
 
-{% do stunnel.lookup.update(salt['grains.filter_by']({
+{% set lookup = salt['pillar.get']('stunnel:lookup', salt['grains.filter_by']({
     'upstart': {
-        'init_file': '/etc/init/stunnel.conf'
+        'init_file': '/etc/init/stunnel.conf',
+        'open_file_limit': '65536'
```

Comment:
I can't see why this should be specific to upstart, can we move it to the initial `pillar.get` block instead.

Reply:
Fixed!